test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MicrosoftDataSqlClientTests.cs

```diff
@@ -0,0 +1,66 @@
+#if !NET452
+using Datadog.Core.Tools;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AdoNet
+{
+    public class MicrosoftDataSqlClientTests : TestHelper
+    {
+        public MicrosoftDataSqlClientTests(ITestOutputHelper output)
+            : base("Microsoft.Data.SqlClient", output)
+        {
+            SetServiceVersion("1.0.0");
+        }
+
+        [Theory]
+        [MemberData(nameof(PackageVersions.MicrosoftDataSqlClient), MemberType = typeof(PackageVersions))]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
+        public void SubmitsTracesWithNetStandard(string packageVersion)
+        {
+            // Note: The automatic instrumentation currently does not instrument on the generic wrappers
+            // due to an issue with constrained virtual method calls. This leads to an inconsistency where
+            // the .NET Core apps generate 4 more spans than .NET Framework apps (2 ExecuteReader calls *
+            // 2 interfaces: IDbCommand and IDbCommand-netstandard).
+            // Once this is fully supported, this will add another 2 complete groups for all frameworks instead
+            // of 4 extra spans on net461 and netcoreapp2.0+
+#if NET452
```

Comment:
You can delete these two lines since the test is not compiled for NET452

Reply:
Since I was already modifying the next couple of lines, I went ahead and implemented this suggestion as well in 5f3b269cab32009d1eaf663f2383e90012ae8fa4

Reply:
Thanks!